service_container.rst

```diff
@@ -1072,6 +1072,48 @@ If you want to pass the second, you'll need to :ref:`manually wire the service <
     and the automatically loaded service will be passed - by default - when you type-hint
     ``SiteUpdateManager``. That's why creating the alias is a good idea.
 
+Injecting the content of YAML file as argument
+----------------------------------------------
+
+.. versionadded:: 4.3
```

Comment:
please add a blank line after the directive, thank you

Reply:
Done

Reply:
Thank you